### PR TITLE
Reword the watcher 'always' and 'never' condition docs

### PR DESCRIPTION
**Pull request by crispybacon**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Comment by crispybacon** [comment]
I have signed the contributor agreement.

File: x-pack/docs/en/watcher/condition.asciidoc [hunk 1]
```diff
@@ -6,16 +6,16 @@
...
 When a watch is triggered, its condition determines whether or not to execute
-the watch actions. {watcher} supports the following condition types: 
+the watch actions. {watcher} supports the following condition types:
 
-* <<condition-always,`always`>>: set the watch condition to `true` so the watch 
+* <<condition-always,`always`>>: a condition whose result is `true`, so the watch
 actions are always executed.
-* <<condition-never,`never`>>: set the watch condition to `false` so the watch
+* <<condition-never,`never`>>: a condition whose result is `false`, so the watch
 actions are never executed.
```
Comment (debadair) [inline x-pack/docs/en/watcher/condition.asciidoc:14]:
```suggestion
* <<condition-never,`never`>>: The condition always evaluates to `false`, so the watch
actions are never executed.
```

File: x-pack/docs/en/watcher/condition/always.asciidoc [hunk 2]
```diff
@@ -5,19 +5,20 @@
 <titleabbrev>Always condition</titleabbrev>
 ++++
 
-Use the `always` condition to set the condition to `true`. This forces the watch 
-actions to be executed unless they are <<actions-ack-throttle,throttled>>. 
+Use the `always` condition so that watch actions will always be
+executed when the watch is triggered, unless they are
+<<actions-ack-throttle,throttled>>.
```
Comment (debadair) [inline x-pack/docs/en/watcher/condition/always.asciidoc:10]:
```suggestion
Use the `always` condition to perform the watch actions whenever the watch is triggered, unless they are
<<actions-ack-throttle,throttled>>.
```

File: x-pack/docs/en/watcher/condition/never.asciidoc [hunk 3]
```diff
@@ -5,10 +5,10 @@
...
 ++++
 
-Use the `never` condition to set the condition to `false`. This means the
-watch actions are never executed when the watch is triggered. The watch input is
-executed, a record is added to the watch history, and the watch execution ends.
-This condition is generally used for testing.
+Use the `never` condition so that watch actions will never be executed
+when the watch is triggered. The watch input is executed, a record is
+added to the watch history, and the watch execution ends. This
+condition is generally used for testing.
```
Comment (debadair) [inline x-pack/docs/en/watcher/condition/never.asciidoc:11]:
```suggestion
Use the `never` condition to skip performing the watch actions whenever the watch is triggered. The watch input is processed, a record is
added to the watch history, and watch execution ends. 
This condition is generally used for testing.
```

**Review by debadair — COMMENTED** [review]
Left a couple suggestions, otherwise LGTM.

**Review by debadair — APPROVED** [review]
LGTM

**Pull request merged**